vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ElementSteps.java

```diff
@@ -71,6 +79,52 @@ public void selectPickerWheelValue(PickerWheelDirection direction, double offset
                                   "offset", offset)));
     }
 
+    /**
+     * Sets the value of the slider the the number. The number must be greater than or equal to 0. Please make
+     * sure that the number you pass does not exceed the right limit of the slider, this may lead to unexpected
+     * failure.
```

Comment:
```suggestion
     * Sets the value of the slider to the number. The number must be greater than or equal to 0. Make
     * sure the passed number does not exceed the right limit of the slider, this may lead to unexpected
     * failure.
```